base/server/upgrade/10.10.0/04-AddProfileCaAuditSigningCert.py

```diff
@@ -0,0 +1,48 @@
+# Authors:
+#     Christina Fu <cfu@redhat.com>
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from __future__ import absolute_import
+import logging
+import os
+
+import pki
+
+logger = logging.getLogger(__name__)
+
+class AddProfileCaAuditSigningCert(pki.server.upgrade.PKIServerUpgradeScriptlet):
```

Comment:
Actually CI failed due to Python formatting. There should be 2 blank spaces to separate the variables and the class. It looks good otherwise.

Reply:
Interesting.  yeah I saw the two blank lines and thought I'd "clean" it up by removing one. ;-).  I'll add that back. thanks!